### PR TITLE
Create ParallelStreamsLoadTest

### PR DESCRIPTION
**Pull request by lumpfish**
Signed-off-by: Simon Rushton <srushton@redhat.com>

### CODE REVIEW

**Comment by lumpfish** [comment]
Needs to be merged at the same time as https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/401
Part of the fix for https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/379

**Review by smlambert — APPROVED** [review]
LGTM @lumpfish - do you have Grinder links to confirm?

**Review by karianna — APPROVED** [review]
LGTM

**Comment by lumpfish** [comment]
There are grinder links in the associated PR https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/401 (the grinders were run with both PRs).

File: system/lambdaLoadTest/playlist.xml [hunk 1]
```diff
@@ -56,48 +54,7 @@
...
-			<group>system</group>
-		</groups>
-		<impls>
-			<impl>openj9</impl>
-			<impl>ibm</impl>
-		</impls>
-		<platformRequirements>os.linux,vm.cmprssptrs</platformRequirements>
-	</test>
-	<test>
-		<testCaseName>JdiTest</testCaseName>
```
Comment (Mesbah-Alam) [inline system/lambdaLoadTest/playlist.xml:82]: @lumpfish - Was JdiTest removed by mistake?

**Comment by llxia** [comment]
- LambdaLoadTest was excluded on xlinux non-compressedrefs on OpenJ9 related to https://github.com/eclipse/openj9/issues/2209 Is this issue resolved?
- JdiTest is removed from this PR. We do not need this test anymore? The related issue is still open https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/63

File: system/lambdaLoadTest/playlist.xml (hunk 1 above)
Comment (lumpfish) [reply system/lambdaLoadTest/playlist.xml:82]: Not by mistake - it does not belong in a lambda test playlist as it tests the java debug interface.  There isn't a 'debugging' playlist, so 'other' looks like the best fit. I'll create a follow on PR.

**Comment by lumpfish** [comment]
>     * LambdaLoadTest was excluded on xlinux non-compressedrefs on OpenJ9 related to [eclipse/openj9#2209](https://github.com/eclipse/openj9/issues/2209) Is this issue resolved?

This PR should resolve all cases where the test was going OOM.  Most occurred on _xl platforms because the test did not specify a big enouigh heap for the ParallelStreamOperationsTest on those platforms, others because the lambda recursive test ran out of stack space on some platforms. See https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/379 for more details.  There are now no restrictions on which platforms / implementations the tests run, other than those imposed by the test 'modes' (as should be the case).

>     * reference for [eclipse/openj9#6475](https://github.com/eclipse/openj9/issues/6475) is removed. If the issue does not exist or test behavior changed, please comment on the issue.

There are a number of issues in multiple repositories which should be resolved by this PR.  I will wait for a few days until the AdoptOpenJDK nightly builds have been run across all releases to make sure the changes are successful before closing / commenting in the issues.

>     * JdiTest is removed from this PR. We do not need this test anymore? The related issue is still open [AdoptOpenJDK/openjdk-systemtest#63](https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/63)

I removed JdiTest because it does not belong in a lambda test playlist as it tests the java debug interface. There isn't a 'debugging' playlist, so 'other' looks like the best fit. I'll create a follow on PR.

**Pull request merged**